### PR TITLE
Refresh network-fortinet-fortiadc-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-fortinet-fortiadc-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiadc-snmp.md [hunk 1]
```diff
@@ -5,106 +5,142 @@ title: Fortinet FortiADC SNMP
...
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de services
+
+| Nom de la règle                                | Description                                                             |
 |:-----------------------------------------------|:------------------------------------------------------------------------|
 | Net-Fortinet-Fortiadc-SNMP-Interface-Name      | Découvre les interfaces réseaux et supervise le statut et l'utilisation |
-| Net-Fortinet-Fortiadc-SNMP-Virtual-Server-Name | Découvre les serveurs virtuels et supervise le statut et l'utilisation  |
+| Net-Fortinet-Fortiadc-SNMP-Virtual-Server-Name |                                                                         |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortiadc-snmp.md:70]:
```suggestion
| Net-Fortinet-Fortiadc-SNMP-Virtual-Server-Name | Découvre les serveurs virtuels et supervise le statut et l'utilisation |
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortiadc-snmp.md [hunk 2]
```diff
@@ -5,106 +5,141 @@ title: Fortinet FortiADC SNMP
...
+| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule |
+
+More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).
+
+#### Service discovery
+
+| Rule name                                      | Description                                                   |
+|:-----------------------------------------------|:--------------------------------------------------------------|
+| Net-Fortinet-Fortiadc-SNMP-Interface-Name      | Discover network interfaces and monitor bandwidth utilization |
+| Net-Fortinet-Fortiadc-SNMP-Virtual-Server-Name |                                                               |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortiadc-snmp.md:69]:
```suggestion
| Net-Fortinet-Fortiadc-SNMP-Virtual-Server-Name | Discover network virtual servers and monitor utilization |
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortiadc-snmp.md [hunk 3]
```diff
@@ -5,106 +5,141 @@ title: Fortinet FortiADC SNMP
...
+* **Net-Fortinet-Fortiadc-SNMP-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Net-Fortinet-Fortiadc-SNMP-custom" label="Net-Fortinet-Fortiadc-SNMP-custom">
+
+| Service Alias   | Service Template                                  | Service Description                                  | Discovery  |
+|:----------------|:--------------------------------------------------|:-----------------------------------------------------|:----------:|
+| Cpu             | Net-Fortinet-Fortiadc-Cpu-SNMP-custom             | Check the rate of utilization of CPU for the machine |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortiadc-snmp.md:29]:
```suggestion
| Cpu             | Net-Fortinet-Fortiadc-Cpu-SNMP-custom             | Check the rate of CPU utilization for the machine |            |
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortiadc-snmp.md [hunk 4]
```diff
@@ -5,106 +5,141 @@ title: Fortinet FortiADC SNMP
...
+| Hardware        | Net-Fortinet-Fortiadc-Hardware-SNMP-custom        | Check hardware                                       |            |
+| Memory          | Net-Fortinet-Fortiadc-Memory-SNMP-custom          | Check the rate of the utilization of memory          |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortiadc-snmp.md:31]:
```suggestion
| Memory          | Net-Fortinet-Fortiadc-Memory-SNMP-custom          | Check the rate of memory utilization      |            |
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortiadc-snmp.md [hunk 5]
```diff
@@ -113,123 +148,493 @@ It brings the following Service Templates:
 
 ### SNMP Configuration
 
-To use this Pack, the SNMP service must be properly configured on your device:
-* https://docs.fortinet.com/product/fortiac/
+To use this pack, the SNMP service must be properly configured on your **Aruba CPPM** equipment according to
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortiadc-snmp.md:151]: Aruba!!!

**Pull request merged**